plasmapy/formulary/collisions.py

```diff
@@ -200,39 +200,40 @@ def Coulomb_logarithm(
 
     Methods
     ---
-    Classical
-        classical Landau-Spitzer approach. Fails for large coupling
+    LS
+        The classical Landau-Spitzer approach. Fails for large coupling
         parameter where Lambda can become less than zero.
-    GMS-1
-        1st method listed in Table 1 of reference [3]
+    LS_min_I ("GMS-1")
+        1st method listed in Table 1 of reference [4].
         Landau-Spitzer, but with interpolated bmin instead of bmin
         selected between deBroglie wavelength and distance of closest
-        approach. Fails for large coupling
-        parameter where Lambda can become less than zero.
-    GMS-2
-        2nd method listed in Table 1 of reference [3]
+        approach. Fails for large coupling parameter where Lambda can
+        become less than zero.
+    LS_full_I ("GMS-2")
+        2nd method listed in Table 1 of reference [4].
         Another Landau-Spitzer like approach, but now bmax is also
         being interpolated. The interpolation is between the Debye
         length and the ion sphere radius, allowing for descriptions
         of dilute plasmas. Fails for large coupling
         parameter where Lambda can become less than zero.
-        3rd method listed in Table 1 of reference [3]
+    LS_clamp ("GMS-3")
+        3rd method listed in Table 1 of reference [4]
         classical Landau-Spitzer fails for argument of Coulomb logarithm
-        Lambda < 0, therefore a clamp is placed at Lambda_min = 2
-    GMS-4
-        4th method listed in Table 1 of reference [3]
+        Lambda < 0, therefore a clamp is placed at Lambda_min = 2.
+    hyp_LS ("GMS-4")
+        4th method listed in Table 1 of reference [4].
         Spitzer-like extension to Coulomb logarithm by noting that
         Coulomb collisions take hyperbolic trajectories. Removes
         divergence for small bmin issue in classical Landau-Spitzer
         approach, so bmin can be zero. Also doesn't break down as
         Lambda < 0 is now impossible, even when coupling parameter is large.
```

Comment:
```suggestion
        :math:`\Lambda < 0`, therefore a clamp is placed at Lambda_min = 2.
    hyp_LS ("GMS-4")
        4th method listed in Table 1 of reference [4].
        Spitzer-like extension to Coulomb logarithm by noting that
        Coulomb collisions take hyperbolic trajectories. Removes
        divergence for small ``bmin`` issue in classical Landau-Spitzer
        approach, so ``bmin`` can be zero. Also doesn't break down as
        :math`\Lambda < 0` is now impossible, even when coupling parameter is large.
```
Some minor formatting suggestions...